huaweicloud/config/endpoints.go

```diff
@@ -766,6 +766,12 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		WithOutProjectID: true,
 		Product:          "NAT",
 	},
+	"organizations": {
+		Name:             "organizations",
+		Version:          "v1",
+		WithOutProjectID: true,
```

Comment:
according to https://developer.huaweicloud.com/endpoint?Organizations, the endpoint of Organizations is  global and without region, so we should  add ```Scope: "global",```